<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Face Recognition</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font-size: 20px;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">margin: 0;
padding: 0;</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QFrame" name="frame_4">
         <property name="styleSheet">
          <string notr="true">max-height: 100px;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="styleSheet">
             <string notr="true">font-size: 36px;</string>
            </property>
            <property name="text">
             <string>Vuelos</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_6">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QTableWidget" name="flightTable">
            <property name="styleSheet">
             <string notr="true">QTableWidget{
	background-color: rgb(255, 255, 255);
	border-radius: 5px;
	padding-left: 5px;
	padding-right: 5px;
	border: 1px solid rgb(223, 223, 223);
}

QHeaderView::section{
	color: rgb(255, 255, 255);
	background-color: rgb(0, 63, 82);
	font-size: 16px;
}</string>
            </property>
            <attribute name="horizontalHeaderDefaultSectionSize">
             <number>167</number>
            </attribute>
            <column>
             <property name="text">
              <string>Destino</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Fecha</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Hora</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Precio</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Acción</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(48, 124, 124);
max-width: 360px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QFrame" name="frame_3">
         <property name="styleSheet">
          <string notr="true">max-height: 200px;
color:#fff;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="styleSheet">
             <string notr="true">
font-size: 36px;</string>
            </property>
            <property name="text">
             <string>Abordar</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Para abordar se necesita comprobar 
 su identidad.</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="identifyYourselfBtn">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(243, 255, 171);
color: rgc(0,0,0);</string>
            </property>
            <property name="text">
             <string>Identificarse en la cámara</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_5">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(35, 107, 102);
border-radius: 5px;
color: #fff;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Datos del boleto</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Destino</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="destinationLbl">
            <property name="text">
             <string>Cancún - México</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Fecha</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="dateDepartureLbl">
            <property name="text">
             <string>14-05-2022</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Hora</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="hourDepartureLbl">
            <property name="text">
             <string>13:00</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Precio</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="priceLbl">
            <property name="text">
             <string>2 600</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>Nombre del pasajero</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="nameLbl">
            <property name="text">
             <string>Maria Rivera</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>34</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
